src/types.ts

```diff
@@ -21,6 +22,40 @@ export interface OrchestrationTrigger extends FunctionTrigger {
     type: "orchestrationTrigger";
 }
 
+/**
+ * Options object provided to `callHttp()` methods on orchestration contexts
+ */
+export interface CallHttpOptions {
+    /**
+     * The HTTP request method.
+     */
+    method: string;
+    /**
+     * The HTTP request URL.
+     */
+    url: string;
```

Comment:
Not 100% sure if that means it would be better to keep the "new" param "body" as "content" as well. @hossam-nasr: can you check in with the wider DF team to confirm if URI is more appropiate?

Reply:
_tomato to-ma-to, url uri._
I won't block on this :)